calculation/seistech_calc/seistech_calc/plots/plt_plotting.py

```diff
@@ -549,3 +592,403 @@ def plot_uhs_branches(
         plt.close()
     else:
         plt.show()
+
+
+def plot_gms_im_distribution(
+    gms_result_data: Dict,
+    save_file: Path = None,
+):
+    """GMS IM distribution plot for each IM
+
+    Parameters
+    ----------
+    gms_result_data: Dict,
+    save_file: Path, optional
+    """
+    plots_data = utils.calculate_gms_im_distribution(gms_result_data)
+
+    plt.figure(figsize=(16, 9))
+
+    def label_maker(im: str):
+        return (
```

Comment:
Don't think its worth defining a function for this if it only used in one location?